ios/FluentUI/Core/ColorProviding.swift

```diff
@@ -100,14 +100,13 @@ private func brandColorOverrides(provider: ColorProviding,
     /// - Parameters:
     ///   - provider: The `ColorProvider2` whose colors should be used for controls in this theme.
     @objc(setColorProvider:)
-    func setColorProvider(_ provider: ColorProviding) {
+    @discardableResult
+    func setColorProvider(_ provider: ColorProviding) -> FluentTheme {
```

Comment:
It seems we update the fluentTheme with `self.fluentTheme = fluentTheme` so why does this function return a `FluentTheme` now?

Reply:
Ah hm, this was part of an older experiment and can probably be removed.